### PR TITLE
Change phpunit to phpunit-core for d8

### PR DESCRIPTION
**Pull request by joshuataylor**
Don't merge this until it's green.

### CODE REVIEW

**Comment by bojanz** [comment]
Could not read "core".
Error: /home/travis/.composer/vendor/lionsad/drupal_ti/runners/phpunit-core/script.sh exited with a failure.

:(

File: .travis.yml [hunk 1]
```diff
@@ -11,10 +11,7 @@ language: php
 sudo: false
 
 php:
-  - 5.5
```
Comment (bojanz) [inline .travis.yml:14]: These should not be removed? We can't test only on 7, it's unstable and has potentially different behavior.

**Comment by bojanz** [comment]
Opened a PR to fix phpunit-core: https://github.com/LionsAd/drupal_ti/pull/52

**Comment by bojanz** [comment]
The drupal_ti PR has landed.

**Comment by joshuataylor** [comment]
Failures are now as a result of actual failures :)

**Comment by bojanz** [comment]
You rock!

**Pull request merged**